ClearKeep/Scenes/Home/Models/GroupModel.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  GroupModel.swift
+//  ClearKeep
+//
+//  Created by NamNH on 17/05/2022.
+//
+
+import Model
+import ChatSecure
+
+struct GroupModel: IGroupModel {
+	var groupId: Int64 = 0
+	var groupName: String = ""
+	var groupAvatar: String = ""
+	var groupType: String = ""
+	var createdBy: String = ""
+	var createdAt: Int64 = 0
+	var updatedBy: String = ""
+	var updatedAt: Int64 = 0
+	var rtcToken: String = ""
+	var groupMembers: [IMemberModel]
+	var isJoined: Bool = false
+	var ownerDomain: String = ""
+	var ownerClientId: String = ""
+	var lastMessage: IMessageModel? = nil
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Home/Models/GroupModel.swift#L25"></span>Initializing an optional variable with nil is redundant.<br />
<code>redundant_optional_initialization</code> <code>GroupModel.swift:25</code></td>
    </tr>
  </tbody>
</table>